gateway/conf/nginx.conf.liquid

```diff
@@ -120,7 +120,8 @@ http {
 
   server {
 
-    access_log {{ access_log_file | default: "/dev/stdout" }} time;
+    set $access_logs_enabled '1';
```

Comment:
Are you sure about the quotes?

Reply:
I think it's correct. From the docs:
`The if parameter (1.7.0) enables conditional logging. A request will not be logged if the condition evaluates to “0” or an empty string.`

Reply:
Yes. But what I meant is that `set` value should not be quoted. Now the value is `'1'`, not `1`.
It is a bit confusing (even though it will still work).

Reply:
Ok. There is some magic going on. 

Both these are the same value (string):

```nginx
    set $access_log_enabled '1';
    set $access_log_enabled_num 1;
```

I thought we had a PR to strip `''` from some of the values because it was generating quotes. My bad I guess.